internal/server/v2/parser.go

```diff
@@ -185,3 +190,29 @@ func ParseLinkedNodes(expr string) (*LinkedNodes, error) {
 	}
 	return g, nil
 }
+
+// Gets the contained in place from an expression.
+func GetContainedInPlace(expr string) (*ContainedInPlace, error) {
```

Comment:
Rename to `ParseContainedInPlace`? This sounds like a data fetch.

Reply:
done